app/views/news/2018-06-07-may-2018-monthly-update.html.md

```diff
@@ -0,0 +1,66 @@
+---
+summary: In May, we paid for 73 hours of developer work, prototyped new metrics rankings for Ruby Toolbox, and added a new flag to a Bundler command.
+---
+
+Hello! Welcome to the monthly update. During May, our work was supported by [GitLab](https://about.gitlab.com/), [Stitch Fix](http://multithreaded.stitchfix.com/), and many others.
+
+## ruby together news
+
+In May, Ruby Together was supported by 77 different companies, including Sapphire member [Stripe](https://stripe.com). [Murb](https://murb.nl/) and [StackPath](https://www.stackpath.com/) joined as Onyx members, and developer Gregory Brown (@practicingdev) became a member. 
+
+We were supported by a total of 86 individual members and 68 friends of Ruby Together. Thanks to all of our members for making everything that we do possible.
+
+In community events news, our developer evangelist [PJ Hagerty](http://twitter.com/aspleenic) spoke at MalmoRB and Ruby Ireland, and visited CopenhagenRB among other Meetups.
```

Comment:
@indirect do we want to add anything here re: GSoC students?

Reply:
for bundler, we've made lots of progress on `bundle remove`!

Reply:
@segiddins yup, looks like @indirect added that at the top of the update! 😄